### DISCUSSION TITLE
V7: error rendering when using useFieldArray

### DISCUSSION THREAD

**Discussion by jonsoku-dev** [react-hook-form/react-hook-form#4840] [Q&A] [answered]
Thank you for developing a good library.
I have a question. See the [sandbox](https://codesandbox.io/s/silly-breeze-oci5q?file=/src/App.tsx) below. 
The problem shown in the above mov occurs.
I want to display the error index correctly. Is there a way? And what is the cause?
Please help me .. !

https://user-images.githubusercontent.com/51524954/115114222-0a00ce80-9fc9-11eb-8bd3-984ed0b283b1.mov

i dont want to remove current validation error. so i cant use 'clearErrors'..

**Comment by leapful** [comment, marked as answer]
@jonsoku2 

The root cause is that validation message is not re-rendered after removing index from useFieldArray which is the expected behavior. You will need to trigger validation by using method "trigger" in order to get the correct validation message.

Working sample:https://codesandbox.io/s/ancient-frost-j10qt

**Reply by jonsoku-dev** [reply]
it works for me :)
Thank you 👍
